.storybook/main.js

```diff
@@ -1,13 +1,15 @@
 const path = require("path");
 
 module.exports = {
-  core: {
-    builder: "webpack5",
-  },
   typescript: {
     reactDocgen: "react-docgen-typescript-plugin",
   },
-  stories: ["../**/*.stories.@(mdx|js|jsx|ts|tsx)"],
+
+  stories: [
+    "../src/**/*.stories.@(mdx|js|jsx|ts|tsx)",
+    "../tokens/**/*.stories.@(mdx|js|jsx|ts|tsx)",
+  ],
+
```

Comment:
Ignore story files copied to `dist/` (icons, for example, just copies everything in src to dist, and that should be okay)